pkg/signup/service/signup_service.go

```diff
@@ -288,6 +288,7 @@ func (s *ServiceImpl) GetSignup(userID string) (*signup.Signup, error) {
 		if err != nil {
 			return nil, errors2.Wrapf(err, "error when retrieving ToolchainStatus to set Che Dashboard for completed UserSignup %s", userSignup.GetName())
 		}
+		signupResponse.ProxyURL = status.Status.HostRoutes.ProxyURL
```

Comment:
Let's add it to our unit tests. And I think we could/should even verify that the proxyURL is set in our e2e tests.